docs/tutorials/deploy_openfaas_workload.md

```diff
@@ -0,0 +1,242 @@
+# OpenFaaS extended workload
```

Comment:
We should reach an agreement on Extended Workload or extended workload.

Reply:
Yeah, I think we should use Extended Workload in title